docs/multiple-users.md

```diff
@@ -0,0 +1,53 @@
+# Multiple users in GCM
+
+We're sometimes asked, "Does GCM support multiple users?" The answer is a bit complex (though ultimately, it's "yes").
+
+## Foundations: Git and Git hosts
+
+Git itself doesn't have a strong concept of "user". There's the `user.name` and `user.email` which get embedded into commit headers/trailers, but these are arbitrary strings. GCM doesn't interact with this notion of a user at all. You can put whatever you want into your `user.*` config, and nothing in GCM will change at all.
+
+Git hosting providers (like GitHub or Bitbucket) _do_ have a concept of "user". Typically it's an identity like a username or email address, plus a password or other credential to perform actions as that user. You may have guessed by now that GCM (the Git **Credential** Manager) does work with this notion of a user.
+
+## People, identities, credentials, oh my!
+
+You (a physical person) may have one or more user accounts (identities) with one or more Git hosting providers. Since Git doesn't really understand what a "user" is, it's not particularly natural to work with multiple identities against a single hosting provider. By default, Git naturally assumes one identity per domain. If you have multiple identites on one domain (or a single identity on multiple domains!), this one-to-one assumption doesn't hold.
```

Comment:
> Since Git doesn't really understand what a "user" is, it's not particularly natural to work with multiple identities against a single hosting provider.

Not true. It's just that it's little used and the host providers (except Bitbucket) don't help you in these scenarios.

Reply:
Fair enough. I was trying to get at the notion of the "empty string" user being default and you having to work to get a user part. But as written, this is basically incorrect.